src/components/MemberArea/model.js

```diff
@@ -102,10 +108,29 @@ export default {
         label: 'LinkedIn',
         prefix: 'https://linkedin.com/in/',
         validate: value => linkedinValidation(value),
+        helpText: <span class="helper-text">Add only your linkedin username</span>
```

Comment:
```suggestion
        helpText: <span class="helper-text">Add only your LinkedIn username</span>
```

Reply:
Also, maybe instead of repeating on `<span class="helper-text">` you can add that class to the `div`.

https://github.com/Coding-Coach/find-a-mentor/blob/8e72cae91800aa08d08c36a96443a0ab825bc0e8/src/components/MemberArea/EditProfile.js#L239